out/notebooks/renderer/components.jsx

```diff
@@ -32,18 +32,19 @@ export default function VizSelector({value, onChange, isCollapsed}) {
         <div>
         {!collapsed &&
         <ButtonGroup tabIndex={-1} style={{"marginBottom": 10}}>
+            {/* Hide certain Viz options for SPL2 to focus use-case on developer audience */}
```

Comment:
This was feedback provided by architecture in order to limit the visualizations to events and table in order to focus the use-case on the developer audience for SPL2, for a full search/viz experience we want to encourage users to use our integrated cloud and data manager experiences